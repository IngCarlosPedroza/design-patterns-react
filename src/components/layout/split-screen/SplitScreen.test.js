import { render } from '@testing-library/react'
import SplitScreen from './SplitScreen'

const Left = () => <p>left</p>
const Right = () => <p>right</p>

it('renders something', () => {
  const { container } = render(
    <SplitScreen leftWeight={1} rightWeight={1}>
      <Left />
      <Right />
    </SplitScreen>)

  expect(container).toMatchSnapshot()
})
