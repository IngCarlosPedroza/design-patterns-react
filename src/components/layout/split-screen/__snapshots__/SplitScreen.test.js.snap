// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders something 1`] = `
<div>
  <div
    class="sc-bdvvtL jxnajB"
  >
    <div
      class="sc-gsDKAQ eSUWIQ"
    >
      <p>
        left
      </p>
    </div>
    <div
      class="sc-gsDKAQ eSUWIQ"
    >
      <p>
        right
      </p>
    </div>
  </div>
</div>
`;
